src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/services/AuthService.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.reform.bulkscanccdeventhandler.services;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.authorisation.validators.AuthTokenValidator;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.services.exception.ForbiddenException;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.services.exception.UnauthenticatedException;
+
+import static java.util.Arrays.asList;
+import static org.apache.commons.lang.StringUtils.isBlank;
+
+@Component
+public class AuthService {
+
+    private final AuthTokenValidator authTokenValidator;
+    private final String[] allowedServices;
+
+    @Autowired
+    public AuthService(
+        AuthTokenValidator authTokenValidator,
+        @Value("${allowed-services}") String[] allowedServices) {
```

Comment:
firstly got confused that it is weirdly constructed but it's just styled: `)` should be in new line ;)

also, `@Autowired` is not needed since you have `@Component`

Reply:
removed `@autowired` and moved `)` to next line